docs/UserGuide.md

```diff
@@ -163,13 +163,14 @@ Examples:
 
 ### Sort persons : `sort`
 
-* Sorts all persons by specified `comparator`.
-* The comparator refers to the attribute(s) provided for the basis to sort.
+* Sorts all persons by specified attribute.
+* Current attributes allowed for sorting: `/byname` for name, `/byemail` for email, `/byphone` for phone, '/byaddress' for address. `/reverse` field to indicate reverse is optional.
```

Comment:
Consider changing the command to not use prefixes with / but instead use just the words themselves

Reply:
We previously discussed about this, will keep the current change first because / is necessary for reverse.